Marlin/Conditionals_LCD.h

```diff
@@ -108,7 +108,26 @@
     #define ULTIPANEL
     #define NEWPANEL
 
-  #endif
+  #elif ENABLED(CR10_stockdisplay)
```

Comment:
Since these are just delays specific to this board when using that display, shouldn't these be (and already are in) pins_MELZI_CREALITY.h ?

https://github.com/MarlinFirmware/Marlin/blob/1.1.x/Marlin/pins_MELZI_CREALITY.h#L60-L63

Reply:
I was looking at something else and saw this, so I think your REVERSE_MENU_DIRECTION might over-rided by Configuration.h without including any ifdef/ifenabled checks (although it mentions only `pins.h` and `Configuration_adv.h`):

```
/**
 * Conditionals_LCD.h
 * Conditionals that need to be set before Configuration_adv.h or pins.h
 */

#ifndef CONDITIONALS_LCD_H // Get the LCD defines which are needed first
```

To be 100% sure, just put your changes into Conditionals_LCD.h, then compile / load on your printer with and without REVERSE_MENU_DIRECTION set in Configuration.h and ensure it changes.  If it does, you should be good so no changes would be needed.

Reply:
See my comment on next code block re: define precedence, which may make this okay to keep as well.

Reply:
I removed the menu direction